Common/Data/Format/IniFile.cpp

```diff
@@ -159,31 +177,40 @@ static std::string EscapeComments(const std::string &value) {
 	return result;
 }
 
+void ParsedIniLine::ParseFrom(const std::string &line) {
+	// TODO: Skip whitespace.
```

Comment:
I suppose it did before, so shouldn't be hard to... my preferred way is to use strspn(), it's basically made for skipping whitespace.

-[Unknown]

Reply:
Interestingly, I've never heard of strspn!